tests/paren_space/jest-test-each.js

```diff
@@ -0,0 +1,8 @@
+test.each`
```

Comment:
TIL

Reply:
Oh yes, `test.each` is a weird feature of Jest 🙂